test/services/database-analyzer/database-analyzer-sequelize.test.js

```diff
@@ -64,5 +64,16 @@ describe('services > database analyser > Sequelize', () => {
       expect(result.rentals).toStrictEqual(expected.rentals);
       await sequelizeHelper.close();
     }, 10000);
+
+    it('should detect snake_case even with no fields in the table', async () => {
```

Comment:
[TESTS] This test only covers one of the two issues. Could you add one that checks for the other one ?

Reply:
The other one is already handled by the diff here : https://github.com/ForestAdmin/lumber/pull/445/commits/dee97a341c778821ebae6a993f3e6284e740bec8#diff-f40ec2acf55e196d0bce15c63549422cL28

The previous test was erroneous as it tested the wrong pattern (snake_case in `through`), when it should have been camelCase. My fix made this test stop passing, renaming the value to camelCase is the actual test.